src/precompiled_tasks/gen/migration.cr

```diff
@@ -1,3 +1,3 @@
 require "../../avram"
 
-Gen::Migration.new.call
+Gen::Migration.new.print_help_or_call(ARGV)
```

Comment:
This makes it so the task prints the help message if given `-h`, `help` or `--help`